src/App.js

```diff
@@ -1,23 +1,60 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import foodsJSON from './foods.json';
+import FoodBox from './Components/FoodBox';
+import { Row, Divider, Button } from 'antd';
+import AddFoodForm from './Components/AddFoodForm';
+import Search from './Components/Search';
 
 function App() {
+  const [foods, setFoods] = useState(foodsJSON);
+  const [showFoods, setShowFoods] = useState(foodsJSON);
+
+
+    const deleteFood = (foodName) => {
+    const filteredFoods = foods.filter((food) => {
+      return food.name !== foodName; 
+    });
+    setFoods(filteredFoods);
+  };
+  
+  const addToList = (food) => {
+    const foodsCopy = [food, ...foodsJSON];
```

Comment:
You are creating a copy of the `foods` array. In the `addToList` function, you are currently using `foodsJSON` which is the original data and not the updated state `foods`. Instead, you should use `foods` to create a copy of the current state and add the new food item to it.

```
const addToList = (food) => {
    const foodsCopy = [food, ...foods];
    setFoods(foodsCopy);
  };
```